### PR TITLE
PRL-6673: Service of Documents

### PR DESCRIPTION
**Pull request by abigailsmith97**
https://tools.hmcts.net/jira/browse/PRL-6673
Name of event: DA case progression - service of documents event

Number of pages: 5

User(s) to complete event: Court Admin

Dependency:

courtNav create case
Court admin has completed "check application" WA
Court admin has completed "send to gatekeeper" WA

Definition of done:

All happy and accessibility journeys are automated.
Jenkins PR build is passing and green.

### CODE REVIEW

File: e2e/tests/global.setup.ts [hunk 1]
```diff
@@ -15,6 +15,18 @@ setup("Setup solicitor user", async ({ page }) => {
 });
 
 setup("Retrieve bearer token for citizen user creation", async () => {
```
Comment (RobL681) [inline e2e/tests/global.setup.ts:17]: Should this be "Retrieve bearer token for courtNav DA case creation"?

File: yarn.lock
[the remark this answers is not in the document]
Comment (abigailsmith97) [reply yarn.lock:1]: no but my pr wouldn't build without changing it...
https://build.hmcts.net/job/HMCTS_j_to_z/job/prl-e2e-tests/job/PR-210/
look at all those sad failures

File: e2e/common/createCaseHelper.ts [hunk 2]
```diff
@@ -13,23 +12,17 @@ async function createDaCitizenCourtNavCase(
...
-    "daCourtNavCreateCase",
-    apiContextDaCreateCase,
-  );
-  if (!tokenDaCreateCase) {
-    throw new Error("Setup failed: Unable to get bearer token.");
-  }
-  process.env.COURTNAV_CREATE_CASE_BEARER_TOKEN = tokenDaCreateCase;
+  const apiContextDaCreateCase2: APIRequestContext = await request.newContext();
+  const tokenDaCreateCase = process.env
+    .COURTNAV_CREATE_CASE_BEARER_TOKEN as string;
```
Comment (RobL681) [inline e2e/common/createCaseHelper.ts:17]: Does COURTNAV_CREATE_CASE_BEARER_TOKEN need to be defined in example.env?

File: e2e/tests/global.setup.ts (hunk 1 above)
Comment (abigailsmith97) [reply e2e/tests/global.setup.ts:17]: ah need to swap my titles rounds, got them mixed up

File: e2e/common/createCaseHelper.ts (hunk 2 above)
Comment (abigailsmith97) [reply e2e/common/createCaseHelper.ts:17]: doesn't need to be defined but is being stored in a env var for use across the tests

**Review by RobL681 — DISMISSED** [review]

**Review by RobL681 — APPROVED** [review]

**Review by Scralfie — APPROVED** [review]

**Pull request merged**